src/actions/render.js

```diff
@@ -3,74 +3,85 @@ const Mustache = require("mustache");
+      return renderLocalization(view, platform, language, outputPath);
+    });
+    return [...acc, ...renderResults];
+  }, []);
 
-      platforms.forEach(platform => {
-        const translationsForPlatform = translationsForLanguage.filter(
-          translation =>
-            [platform, platformKeywords.SHARED].includes(translation.platform)
-        );
-        const view = viewForPlatform(translationsForPlatform, platform);
-        const renderedView = renderPlatform(
-          view,
-          platform,
-          language,
-          outputPath
-        );
-        acc.push(renderedView);
-      });
-      return acc;
-    }, [])
-    .reduce((acc, result) => {
-      return acc.flatMap(value => result.map(r => value.concat([r])));
-    }, Result.success([]));
-};
-
-const renderPlatform = (view, platform, language, basePath) => {
+  const codeGenerationRenders = platforms
+    .map(platform => [createCodeGenView(translations, platform), platform])
+    .map(([view, platform]) => renderCodeGenView(view, platform, outputPath));
+
+  return [...localizationRenders, ...codeGenerationRenders]
+    .filter(render => !!render)
+    .reduce(
+      (acc, result) => acc.flatMap(value => result.map(r => value.concat([r]))),
+      Result.success([])
+    );
+};
+
+const includeInPlatform = (translation, platform) =>
+  [platform, platformKeywords.SHARED].includes(translation.platform);
+
+const renderLocalization = (view, platform, language, basePath) => {
+  const outputPath = localizationOutputPath(basePath, platform, language);
   view.lowerCase = () => (text, render) => render(text).toLowerCase();
-  return templateForPlatform(platform)
+  return localizationTemplate(platform)
     .map(template => Mustache.render(template, view))
-    .map(data => {
-      return {
-        path: outputPathForPlatform(basePath, platform, language),
-        data: data
-      };
-    });
+    .map(render => ({ path: outputPath, data: render }));
 };
 
-const templateForPlatform = platform => {
+const renderCodeGenView = (view, platform, basePath) => {
+  const outputPath = codeGenerationOutputPath(basePath, platform);
+  return codeGenerationTemplate(platform)
+    .map(({ template, partials }) => Mustache.render(template, view, partials))
+    .map(render => ({ path: outputPath, data: render }));
+};
+
+const localizationTemplate = platform => {
   switch (platform) {
     case platformKeywords.ANDROID:
-      return loadFile(
-        path.resolve(__dirname, "../../templates/strings_xml_file.mustache")
-      );
+      return loadFile(path.resolve(__dirname, "../../templates/strings_xml_file.mustache"));
+    case platformKeywords.IOS:
+      return loadFile(path.resolve(__dirname, "../../templates/localizable_strings_file.mustache"));
+  }
+};
+
+const codeGenerationTemplate = platform => {
+  switch (platform) {
+    case platformKeywords.ANDROID:
+      return Result.success({ template: "", partial: null });
```

Comment:
Are we going to add Android Code generation in the (near) future? If not, I'm not sure I like this approach of returning a `Result.success` with a `null` payload?